<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillForm</class>
 <widget class="QWidget" name="BillForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>655</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="flashMsgUp">
         <property name="text">
          <string>Error Notification</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>530</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>582</x>
            <y>60</y>
            <width>96</width>
            <height>30</height>
           </rect>
          </property>
          <property name="text">
           <string>Invoice No</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>19</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Customer Code</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>279</x>
            <y>30</y>
            <width>105</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>Customer Address</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditProductCode">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>325</y>
            <width>91</width>
            <height>23</height>
           </rect>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditProductName">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>325</y>
            <width>321</width>
            <height>23</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditTotal">
          <property name="geometry">
           <rect>
            <x>710</x>
            <y>325</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditRate">
          <property name="geometry">
           <rect>
            <x>620</x>
            <y>325</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditUnit">
          <property name="geometry">
           <rect>
            <x>530</x>
            <y>325</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditQty">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>325</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditInvoiceNo">
          <property name="geometry">
           <rect>
            <x>682</x>
            <y>60</y>
            <width>101</width>
            <height>23</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditCustomerCode">
          <property name="geometry">
           <rect>
            <x>122</x>
            <y>30</y>
            <width>150</width>
            <height>23</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>22</y>
            <width>96</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Invoice Date</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>60</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Customer Name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditCustomerName">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>60</y>
            <width>151</width>
            <height>23</height>
           </rect>
          </property>
         </widget>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>-3</x>
            <y>352</y>
            <width>802</width>
            <height>10</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="Line" name="line_2">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>357</y>
            <width>20</width>
            <height>173</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="Line" name="line_3">
          <property name="geometry">
           <rect>
            <x>470</x>
            <y>356</y>
            <width>20</width>
            <height>173</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>360</y>
            <width>211</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font : 15px;
font: bold;</string>
          </property>
          <property name="text">
           <string>Customer Credit</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>500</y>
            <width>57</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Balance</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTableView" name="tableViewCustomerBalance">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>364</y>
            <width>251</width>
            <height>129</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>490</x>
            <y>368</y>
            <width>51</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Tax</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>490</x>
            <y>470</y>
            <width>51</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>Change</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditTax">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>365</y>
            <width>71</width>
            <height>23</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditChange">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>467</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditBalance">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>500</y>
            <width>101</width>
            <height>23</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonClear">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>500</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonSave">
          <property name="geometry">
           <rect>
            <x>600</x>
            <y>500</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonDelete">
          <property name="geometry">
           <rect>
            <x>700</x>
            <y>500</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditCustomerAddress">
          <property name="geometry">
           <rect>
            <x>389</x>
            <y>27</y>
            <width>191</width>
            <height>60</height>
           </rect>
          </property>
         </widget>
         <widget class="QDateEdit" name="dateEditInvoiceDate">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>681</x>
            <y>30</y>
            <width>103</width>
            <height>24</height>
           </rect>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="accelerated">
           <bool>false</bool>
          </property>
          <property name="keyboardTracking">
           <bool>true</bool>
          </property>
          <property name="displayFormat">
           <string>dd/MM/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="timeSpec">
           <enum>Qt::LocalTime</enum>
          </property>
         </widget>
         <widget class="Line" name="line_5">
          <property name="geometry">
           <rect>
            <x>-9</x>
            <y>356</y>
            <width>20</width>
            <height>173</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="Line" name="line_6">
          <property name="geometry">
           <rect>
            <x>789</x>
            <y>356</y>
            <width>20</width>
            <height>173</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonPrint">
          <property name="geometry">
           <rect>
            <x>390</x>
            <y>500</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Print</string>
          </property>
         </widget>
         <widget class="Line" name="line_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>524</y>
            <width>800</width>
            <height>10</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>629</x>
            <y>391</y>
            <width>60</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Round Off</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_19">
          <property name="geometry">
           <rect>
            <x>633</x>
            <y>470</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Given</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_18">
          <property name="geometry">
           <rect>
            <x>521</x>
            <y>435</y>
            <width>171</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Too Be Paid</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>628</x>
            <y>368</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Total</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>484</x>
            <y>401</y>
            <width>59</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Discount</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditGrandTotal">
          <property name="geometry">
           <rect>
            <x>699</x>
            <y>365</y>
            <width>91</width>
            <height>23</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditRoundOff">
          <property name="geometry">
           <rect>
            <x>698</x>
            <y>398</y>
            <width>91</width>
            <height>23</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditTooBePaid">
          <property name="geometry">
           <rect>
            <x>699</x>
            <y>432</y>
            <width>91</width>
            <height>23</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditGiven">
          <property name="geometry">
           <rect>
            <x>700</x>
            <y>467</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditDiscount">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>398</y>
            <width>71</width>
            <height>23</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditLimit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>406</y>
            <width>113</width>
            <height>30</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditUsed">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>447</y>
            <width>113</width>
            <height>30</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditAvailable">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>487</y>
            <width>113</width>
            <height>30</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>11</x>
            <y>449</y>
            <width>71</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>Used</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>406</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Limit</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>26</x>
            <y>489</y>
            <width>56</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Available</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTableWidget" name="tableWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>780</width>
            <height>231</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>780</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditInvoiceNo</tabstop>
  <tabstop>dateEditInvoiceDate</tabstop>
  <tabstop>lineEditCustomerCode</tabstop>
  <tabstop>lineEditCustomerName</tabstop>
  <tabstop>lineEditCustomerAddress</tabstop>
  <tabstop>lineEditProductCode</tabstop>
  <tabstop>lineEditProductName</tabstop>
  <tabstop>lineEditQty</tabstop>
  <tabstop>lineEditUnit</tabstop>
  <tabstop>lineEditRate</tabstop>
  <tabstop>lineEditTotal</tabstop>
  <tabstop>lineEditTax</tabstop>
  <tabstop>lineEditChange</tabstop>
  <tabstop>lineEditBalance</tabstop>
  <tabstop>pushButtonClear</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonDelete</tabstop>
  <tabstop>lineEditGrandTotal</tabstop>
  <tabstop>lineEditDiscount</tabstop>
  <tabstop>lineEditRoundOff</tabstop>
  <tabstop>lineEditTooBePaid</tabstop>
  <tabstop>lineEditGiven</tabstop>
  <tabstop>tableViewCustomerBalance</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
